### DISCUSSION TITLE
eth0.me flagged as malicious by multiple providers

### DISCUSSION THREAD

**Discussion by Brophet** [telekom-security/tpotce#1285] [General]
Hi, I set up tpot and found that the url eth0.me was blocked by my DNS, also the URL is marked as malicious by several sites. As far as I could find out, this URL is called in myip.sh, so I have temporarily removed the URL from myip.sh but wanted to point out my finding.
In addition, I noticed that a few of the URLs contained in myip.sh are sporadically marked on Virustotal and sometimes have a surprisingly bad community score, so I thought I'd throw that out there.
If this is a known problem I would appreciate a confirmation.
With kind regards

Eth0.me on virustotal
![image](https://user-images.githubusercontent.com/30075556/226887712-663780be-8db0-4525-aa63-33ca51d49ba4.png)

Eth0.me on alienvault
![image](https://user-images.githubusercontent.com/30075556/226888000-b1216e0b-89b8-4f68-8b39-390d5b114ce2.png)

api.ipify.org community score on virustotal
![image](https://user-images.githubusercontent.com/30075556/226891236-288c8eef-fe05-4bf1-a9cd-d8839e08d547.png)

**Comment by t3chn0m4g3** [comment]
By that logic the Google DNS servers would need to be classified as [malicious](https://otx.alienvault.com/indicator/ip/8.8.8.8) as well, but were whitelisted, yet still millions of malicious files / scripts use them. The same accounts basically for services like eth0.me and others which determine the public ip address but have no sponsor for whitelisting them. Ipify in particular uses rate limits which make it less interesting for threat actors to use it. Run a simple curl (as myip does) against the services, only the public IP is returned.

**Reply by Brophet** [reply]
Thank you for answering my question and creating this great project. I had suspected that the warning comes from the fact that many malicious actors use the service, but I was not sure. I am working on a student project with this project and it helps me a lot to have this confirmed.
